linera-client/src/client_options.rs

```diff
@@ -1099,10 +1099,15 @@ pub enum NetCommand {
         #[arg(long, default_value = "grpc")]
         external_protocol: String,
 
+        /// If present, a faucet is started using the chain provided by --faucet-chain, or the
+        /// default chain if not provided.
```

Comment:
I don't get it. We just created a local test network and an admin wallet. The default chain is root chain 0.

Reply:
Oh ok so you want `--faucet` instead of `--with-faucet-chain 0` but we're dealing with chain IDs here so the API was never going to be standard.